modules/apps/portal-security-sso/portal-security-sso-openid-connect-impl/src/main/java/com/liferay/portal/security/sso/openid/connect/internal/configuration/admin/service/OpenIdConnectProviderManagedServiceFactory.java

```diff
@@ -476,12 +481,16 @@ private void _updateOAuthClientEntry(
 
 		try {
 			String authServerWellKnownURI = _updateOAuthClientASLocalMetadata(
-				defaultUserId, properties);
+				defaultUserId, oldProperties);
```

Comment:
I think passing `oldProperties` to this function will make it never do some changes/updates.